config/DynamicEvaluator/checker_phpstan_include_project.neon

```diff
@@ -1,5 +1,3 @@
-includes:
```

Comment:
Revert functionality related to the project settings for php stan

Reply:
++